webui.py

```diff
@@ -81,6 +84,7 @@ def generate_clicked(*args):
     css=modules.html.css).queue()
 
 with shared.gradio_root:
+    currentTask = gr.State(worker.AsyncTask(args=[]))
```

Comment:
fyi this can also be empty, just to prevent future checks for e.g. processing or last_stop